### PR TITLE
Add configuration files

### PR DESCRIPTION
**Pull request by nappex**

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Config
+
+config :onigumo, :http_client, HTTPoison
```
Comment (Glutexo) [inline config/dev.exs:3]: Why don’t we use parentheses on function calls?

```suggestion
config(:onigumo, :http_client, HTTPoison)
```

File: config/dev.exs [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Config
```
Comment (Glutexo) [inline config/dev.exs:1]:
```suggestion
import(Config)
```

File: config/test.exs [hunk 3]
```diff
@@ -0,0 +1,3 @@
+import Config
+
+config :onigumo, :http_client, HTTPoisonMock
```
Comment (Glutexo) [inline config/test.exs:3]:
```suggestion
config(:onigumo, :http_client, HTTPoisonMock)
```

File: config/test.exs [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import Config
```
Comment (Glutexo) [inline config/test.exs:1]:
```suggestion
import(Config)
```

File: config/config.exs [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import Config
```
Comment (Glutexo) [inline config/config.exs:1]:
```suggestion
import(Config)
```

**Review by Glutexo — APPROVED** [review]
:shipit:

File: config/dev.exs (hunk 1 above)
Comment (nappex) [reply config/dev.exs:3]: nepruď

**Pull request merged**